test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/IntegrationTests.cs

```diff
@@ -58,11 +61,16 @@ public void TraceExportResultIsSuccess(OtlpExportProtocol protocol, string endpo
                 AppContext.SetSwitch("System.Net.Http.SocketsHttpHandler.Http2UnencryptedSupport", true);
             }
 #endif
+            using EventWaitHandle handle = new ManualResetEvent(false);
 
             var exporterOptions = new OtlpExporterOptions
             {
                 Endpoint = new Uri($"http://{CollectorHostname}{endpoint}"),
                 Protocol = protocol,
+                BatchExportProcessorOptions = new()
+                {
+                    ScheduledDelayMilliseconds = 10000,
```

Comment:
minor: given this is used later, maybe use a local const?

Reply:
Updated